packages/clay-tooltip/src/TooltipProvider.tsx

```diff
@@ -66,6 +66,20 @@ const reducer = (state: IState, {type, ...payload}: IAction): IState => {
 	}
 };
 
+function closestParent(node: HTMLElement, s: string) {
```

Comment:
This should probably be called `closestAncestor` or even just `closest` (seeing as that is what you're polyfilling). The word "parent" implies an immediate parent-to-child relationship.